UnityProject/Assets/Resources/ScriptableObjects/Emotes/basicGenderedEmotes/Scream.asset

```diff
@@ -0,0 +1,22 @@
+%YAML 1.1
+%TAG !u! tag:unity3d.com,2011:
+--- !u!114 &11400000
+MonoBehaviour:
+  m_ObjectHideFlags: 0
+  m_CorrespondingSourceObject: {fileID: 0}
+  m_PrefabInstance: {fileID: 0}
+  m_PrefabAsset: {fileID: 0}
+  m_GameObject: {fileID: 0}
+  m_Enabled: 1
+  m_EditorHideFlags: 0
+  m_Script: {fileID: 11500000, guid: 1a1e9a7021acd5343ab7777f5bdc00ff, type: 3}
+  m_Name: Scream
+  m_EditorClassIdentifier: 
+  emoteName: 
+  viewText: did something!
```

Comment:
```suggestion
  viewText: Screams!
```
I imagine that what it should be.

Reply:
odd, why didn't the viewText get commited?